HW 10/Hash table.h

```diff
@@ -0,0 +1,37 @@
+﻿#pragma once
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+//a wrapper for Node
```

Comment:
```suggestion
// a wrapper for Node
```
Надо пробел после символа комментария. А лучше комментарий писать с заглавной и с точкой в конце.